broker/src/main/java/io/zeebe/broker/transport/backpressure/CommandRateLimiter.java

```diff
@@ -14,10 +14,12 @@
 import java.util.Map;
 import java.util.Objects;
 import java.util.Optional;
+import java.util.Set;
 import java.util.concurrent.ConcurrentHashMap;
 
 public class CommandRateLimiter extends AbstractLimiter<Intent> implements RequestLimiter<Intent> {
 
+  private static final Set WHITE_LISTED_COMMANDS = Set.of(JobIntent.COMPLETE, JobIntent.FAIL);
```

Comment:
You can use an `EnumSet<Intent>` here I think, which is optimized for this particular use case. Also better use `Set<Intent>` as the field type.

i.e. 
```java
private static final Set<Intent> WHITE_LISTED_COMMANDS = EnumSet.of(
  JobIntent.COMPLETE,
  JobIntent.FAIL,
  WorkflowInstanceIntent.CANCEL
);
```